bracket-push/bracket-push_test.go

```diff
@@ -29,6 +33,10 @@ var testCases = []struct {
 		expected: true,
 	},
 	{
+		input:    "{[}]",
+		expected: false,
```

Comment:
please do sanity check for me so that we agree whether the result should be true or false for this. I argue it should be false because https://github.com/exercism/x-common/blob/master/bracket-push.md says **nested correctly** and `{[}]` is an incorrect nesting.